samples/excel/85-preview-apis/worksheet-page-layout.yaml

```diff
@@ -0,0 +1,198 @@
+order: 5
+id: excel-worksheet-page-layout
+name: Page layout
+description: Sets the page layout for printing a worksheet.
```

Comment:
If you change the snippet name, consider also changing the description accordingly, e.g. -> "Changes page layout and other settings for printing a worksheet."